### DISCUSSION TITLE
La posiblidad de agregar archivos .omf como extensión adicional.

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#888] [3. Ideas & Feature Requests]
Hola

Nos gustaría tener la posiblidad de poder cargar archivos con extensión .omf. Un archivo OMF tiene esas siglas porque proviene de las siglas de Open Media Framework. Esto no es más que un formato de archivo destinado a transferir medios digitales entre software diferente.

Un cliente lo utiliza para subir dichos archivos, será muy bueno ue se incluya para la componente Files.

Gracias

**Comment by acedward** [comment]
@NicolasPLG creo que deberíamos abrir esto a todos los tipos de archivos, y mostrarle un warning al usuario en casos problemáticos.

**Reply by NicolasPLG** [reply]
aa perfecto me parece muy bien, no quise expandir las extensiones porque no sabría si existen algunas con inconvenientes( ejemplo, txt, zip, rar, etc). 

Gracias @acedward

**Reply by acedward** [reply]
@NicolasPLG tienes un caso donde necesiten esto?

**Reply by NicolasPLG** [reply]
Lo necesitan para subir archivos de apoyos para la edición de videos con audio.
Te dejo lo siguiente:

1. CompanyId: 5ee8c9742712b4000861b6a8
2. link archivi omf: https://drive.google.com/file/d/1SroQLi-6_H2l2eQewo7frSRUSnlsXMGQ/view?usp=sharing

si requieres más detalle me comentas
